sr_utilities/scripts/sr_utilities/hand_finder.py

```diff
@@ -141,6 +141,9 @@ def wait_for_hand_params(self, timeout_in_secs):
             self._hand_h = True
             self._hand_h_parameters = rospy.get_param("/fh_hand")
 
+    def get_hand_h_parameters(self):
+        return self._hand_h_parameters
```

Comment:
I would add:
```
  if not self._hand_h:
      rospy.logerr("No Hand H present - can't get hand parameters")
```
to keep consistency.
Also remove 'object' from HandConfig(object) pls!

Reply:
Good point, added logerr and removed objects :)